hedera-node/hedera-app/src/test/java/com/hedera/node/app/authorization/PrivilegesVerifierTest.java

```diff
@@ -0,0 +1,678 @@
+ */
+
+package com.hedera.node.app.authorization;
+
+import static com.hedera.node.app.service.mono.txns.auth.SystemOpAuthorization.AUTHORIZED;
+import static com.hedera.node.app.service.mono.txns.auth.SystemOpAuthorization.IMPERMISSIBLE;
+import static com.hedera.node.app.service.mono.txns.auth.SystemOpAuthorization.UNAUTHORIZED;
+import static com.hedera.node.app.service.mono.txns.auth.SystemOpAuthorization.UNNECESSARY;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import com.google.protobuf.ByteString;
+import com.google.protobuf.InvalidProtocolBufferException;
+import com.hedera.hapi.node.base.HederaFunctionality;
+import com.hedera.node.app.authorization.Authorizer.SystemPrivilege;
+import com.hedera.node.app.service.mono.pbj.PbjConverter;
+import com.hedera.node.app.service.mono.txns.auth.SystemOpAuthorization;
+import com.hedera.node.app.service.mono.utils.accessors.SignedTxnAccessor;
+import com.hedera.node.config.ConfigProvider;
+import com.hedera.node.config.VersionedConfigImpl;
+import com.hedera.node.config.testfixtures.HederaTestConfigBuilder;
+import com.hederahashgraph.api.proto.java.AccountID;
+import com.hederahashgraph.api.proto.java.ContractDeleteTransactionBody;
+import com.hederahashgraph.api.proto.java.ContractID;
+import com.hederahashgraph.api.proto.java.ContractUpdateTransactionBody;
+import com.hederahashgraph.api.proto.java.CryptoCreateTransactionBody;
+import com.hederahashgraph.api.proto.java.CryptoDeleteTransactionBody;
+import com.hederahashgraph.api.proto.java.CryptoUpdateTransactionBody;
+import com.hederahashgraph.api.proto.java.EthereumTransactionBody;
+import com.hederahashgraph.api.proto.java.FileAppendTransactionBody;
+import com.hederahashgraph.api.proto.java.FileDeleteTransactionBody;
+import com.hederahashgraph.api.proto.java.FileID;
+import com.hederahashgraph.api.proto.java.FileUpdateTransactionBody;
+import com.hederahashgraph.api.proto.java.FreezeTransactionBody;
+import com.hederahashgraph.api.proto.java.SystemDeleteTransactionBody;
+import com.hederahashgraph.api.proto.java.SystemUndeleteTransactionBody;
+import com.hederahashgraph.api.proto.java.Transaction;
+import com.hederahashgraph.api.proto.java.TransactionBody;
+import com.hederahashgraph.api.proto.java.TransactionID;
+import com.hederahashgraph.api.proto.java.UncheckedSubmitBody;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+// This test may look a little weird without context. The original test in mono is very extensive. To ensure that
+// we don't break anything, I copied the test from mono and hacked it a little to run it with the new code.
+// (It was a good thing. I discovered two bugs...) :)
+class PrivilegesVerifierTest {
+
+    private Wrapper subject;
+
+    private static class Wrapper {
```

Comment:
What do we gain using a wrapper here rather than directly calling methods for the class we intend to test?  It seems like unnecessary additional complexity (and opportunity to unintentionally skew results).